base_images/r/docker/Dockerfile

```diff
@@ -17,6 +17,7 @@ WORKDIR /projects
 RUN sed -i -e 's/\/root/\/projects/g' /etc/passwd
 
 RUN conda install -y -c conda-forge mamba=1.3.1
+RUN conda install -y -c conda-forge pyOpenSSL=23.2.0
```

Comment:
can this install be done on the same line as mamba ? 
`
conda install -y -c conda-forge mamba=1.3.1 pyOpenSSL=23.2.0
`

Reply:
No because trying to install it fails with conda: https://repo.dit.maap-project.org/root/maap-workspaces/-/jobs/5040

Reply:
Oh, wait sorry I forgot to merge changes back to pip after testing that conda didn't work... one second

Reply:
Changed back to pip

Reply:
Okay. I think its because you are installing mamba first and then openssl thats causing this to break. Try the reverse. 
`
RUN conda install -y -c conda-forge pyOpenSSL=23.2.0 && \
         conda install -y -c conda-forge mamba=1.3.1 
`